### PR TITLE
Microservices For the Win

### PR DESCRIPTION
**Pull request by nelsongomes**
# Micro services For the Win
- Speaker   : Nelson Gomes
- Available : October 7th and 8th 
- Length    : 30 to 60 minutes depending of available slots
- Language  : English
## Description

Another talk about dos and dont’s of Microservices.
Having a REST API is not the same thing of having a Microservice.
A Microservice can scale, be distributed and needs to handle several types of problems only distributed systems have, behaving like a cluster and with the need to be aware of the state of it’s resources.

So I propose a practical view showing how to make the most of existing frameworks and providing at the end with the source code for the demo for those eager to try new things.

Overview:
- Setting up a Jetty/Jersey server;
- Database access made easy (Yank/DbUtils/HikariCP);
- Using Swagger to generate clients easily;
- Converting a simple REST API into a Microservices distributed system (Hazelcast);
- Adding a cache layer (Google Guava/Hazelcast);
- Circuit breakers and resource management;
- Distributed processing and redundancy;

---
## Speaker Bio

With 16 years of experience on IT, several of them working with open source technologies and projects, I developed a good knowledge of IoT, and on system architechure. 

Worked in SAPO for 6 years, developing the ad serving solution based on OpenX that provided ads for the web, mobile, tv box, phone, email among others targets.

With a degree in Computer Sciences from Faculty of Science from University of Lisbon, as more than 16 years of professional experience in web and app development and worked in several different industries.

Today works in large web projects as project leader, developer, systems architect.

Part of Codebits Programming Contest 2010 8th place winning team.

ITIL Foundation V3 certified, Zend Certified Engineer, Linux LPIC-1 certified
## Links
- Linkedin: https://pt.linkedin.com/in/nelsonricardogomes
## Contact

nelson.ricardo.gomes@gmail.com

### CODE REVIEW

**Comment by cdeath** [comment]
hey @nelsongomes!

the template file should not have been deleted.
can you fix it in another commit?

thanks!

**Comment by nelsongomes** [comment]
Done

File: microservices-ftw.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Micro services For the Win========================
+
+* Speaker   : Nelson Gomes
+* Available : October 7th and 8th 
+* Length    : 30 to 60 minutes depending of available slots
+* Language  : English
```
Comment (dreispt) [inline microservices-ftw.md:6]: Should this be changed to  `Language: Java` ? 😉

Comment (nelsongomes) [inline microservices-ftw.md:6]: The language refers to the language the presentation is given and not the programming language.

Comment (dreispt) [inline microservices-ftw.md:6]: Of course. I just couldn't resist the joke, no pun intended.

**Comment by celso** [comment]
@nelsongomes check your email please

**Comment by nelsongomes** [comment]
Done that thanks.

**Pull request merged**